content/en/graphing/functions/rollup.md

```diff
@@ -5,17 +5,40 @@ kind: documentation
 
-Note that rollups should usually be avoided in [monitor][1] queries, because of the possibility of misalignment between the rollup interval and the evaluation window of the monitor. The start and end of rollup intervals are aligned to UNIX time, not to the start and end of monitor queries, which means that a monitor may evaluate (and trigger on) an incomplete rollup interval containing only a small sample of data. To avoid this issue, users should delay the evaluation of the monitor by (at least) the length of the rollup interval.
+The function takes two parameters, `<METHOD>` and `<TIME>`: `.rollup(<METHOD>,<TIME>)`.
 
-## .as_count() or as_rate()
+| Parameter  | Description                                                                                                     |
+|------------|-----------------------------------------------------------------------------------------------------------------|
+| `<METHOD>` | Can be `sum`/`min`/`max`/`count`/`avg` and defines how data points are aggregated within a given time interval. |
+| `<TIME>`   | Time in second of the interval between two data points displayed.                                               |
 
-These functions are only intended for metrics submitted as rates or counters via StatsD. These functions have no effect for other metric types. For more details about how to use `.as_count()` and `.as_rate()` see [our blog post][2].
+You can use either one individually, or both together like `.rollup(sum,120)` for instance. Find below bar graph displaying a week's worth of cpu usage for a host without using the `.rollup()` function:
 
-Note: [The only available query with `as_count()` is `sum()`][3] (unless using a rollup summary), which is the only mathematical accurate function with such behavior. Learn more with the [detailed documentation about in-application modifiers][4].
+{{< img src="graphing/functions/rollup/smooth_1.png" alt="smooth_1" responsive="true" style="width:60%;" >}}
+
+And now it is the same metric, graphed using a day-long rollup with `.rollup(86400)`:
+
+{{< img src="graphing/functions/rollup/smooth_2.png" alt="smooth_2" responsive="true" style="width:60%;" >}}
+
+## Maximum number of points displayed
+
+When graphing, Datadog imposes a limit of 350 points per graph. In order to respect this limit, Datadog rolls up data points automatically with the `avg` method, effectively displaying the average of all data points within a time interval for a given metric.
+
+If a custom `.rollup()` function is applied and is lower than the Datadog limit, the Datadog limit is used instead. For example, if you're requesting `.rollup(20)` for a month-long window, data is returned at a rollup greater than 20 seconds in order to prevent returning more than 350 points.
```

Comment:
> If a custom `.rollup()` function is applied and is lower than the Datadog limit, the Datadog limit is used instead. For example, if you're requesting `.rollup(20)` for a month-long window, data is returned at a rollup greater than 20 seconds in order to prevent returning more than 350 points.


Let's insert a line before this to clarify. Something to the effect of (this isn't the cleanest copy, just a suggestion):

"A custom `.rollup()` function can be used to set the type of time aggregation applied (avg, min, max, count, sum) and the time interval to rollup. However, if a custom `.rollup()` function is applied and uses a smaller interval than the Datadog limit, the Datadog limit is used instead, while still using the specified rollup method.